pkg/prowgen/jobbase_test.go

```diff
@@ -0,0 +1,418 @@
+package prowgen
```

Comment:
Huge file but really just test scenarios - the actual test code is usually super simple.